f2/src/components/formik/textArea/index.js

```diff
@@ -13,6 +13,7 @@ export const TextArea = ({ field, form, ...props }) => {
         className={
           'open-text-entry' + (largeTextArea ? ' large-text-area' : '')
         }
+        {...props}
```

Comment:
Why are we modifying the TextArea component? What properties are we trying to pass to Form.Control that are not present?

Reply:
"rows", to adjust the height of the textArea. I know you add 'large-text-area', but that is a fixed height (300px). It is more flexible to pass props so that you can define any height for the textArea by "rows", and potentially in the future, we could pass in other props for the textArea to make this component more flexible.